src/rest-server/src/models/v2/job/k8s.js

```diff
@@ -282,6 +282,15 @@ const generateTaskRole = (taskRole, labels, config) => {
     gangAllocation = 'false';
     retryPolicy.fancyRetryPolicy = true;
   }
+  // calculate pod priority
+  // reference: https://github.com/microsoft/pai/issues/3704
+  let jobPriority = 127;
+  if (launcherConfig.enabledHived) {
+    jobPriority = 126 - parseInt(config.taskRoles[taskRole].hivedPodSpec.priority);
```

Comment:
>priority [](start = 73, length = 8)

What if user does not specify hivedPodSpec.priority?

Reply:
default `hivedPodSpec.priority` is `null`, so default `jobPriority = 126 - parseInt(null)` equals to `126`.

Reply:
isn't 126 too high?

Reply:
Can we explictly specify to default as 0? Max=126 Min=-1 And doc it.

Reply:
updated comment in hived priority class